### ISSUE TITLE
fix arc parsing wrt flags

### ISSUE THREAD

**Issue by Pomax** [Pomax/svg-to-something#5]
SVG: `a 5.5 5.5 0 100 11 z` must follow the grammar for {letter, number, number, number, flag, flag, number, number}, which means the 100 component should be treated as "flag:1 flag:0 number:0 number:11".

From the spec:

```
elliptical_arc::=
    ( "A" | "a" ) wsp* elliptical_arc_argument_sequence

elliptical_arc_argument_sequence::=
    elliptical_arc_argument
    | (elliptical_arc_argument comma_wsp? elliptical_arc_argument_sequence)

elliptical_arc_argument::=
    number comma_wsp? number comma_wsp? number comma_wsp
    flag comma_wsp? flag comma_wsp? coordinate_pair

...

sign::= "+"|"-"
number ::= ([0-9])+
flag::=("0"|"1")
comma_wsp::=(wsp+ ","? wsp*) | ("," wsp*)
wsp ::= (#x9 | #x20 | #xA | #xC | #xD)
```

Target: normalization should turn this into `A 5.5 5.5 0 1 0 0 11 Z`

**Comment by thednp** [comment]
I have the solution, [check this out](https://thednp.github.io/svg-path-commander) man, sorry for the delay.

**Comment by Pomax** [comment]
I'm sure you didn't mean it this way, but: what do you want me to take home from that comment? I wasn't particularly unclear on _how_ to implement this. Instead, as I explained before, I had _no justification_ to believe your claim, until I found the spec that backed it up. At which point I filed this issue so that once I have time again, I may revisit it. It hasn't been sitting here not getting worked on because I'm somehow unclear on what I'd need to do: the mix of `number` and `flag` values for `A` mean the parsing needs a forward string parse rather than blind "move 7 terms".

However, I'm in the middle of an incredibly involved and time consuming full rewrite of https://pomax.github.io/bezierinfo's tech stack, including every single piece of example code in that book, so I don't have any more time to spend on updating this repo: I added base support for `A` because I had some spare time: I no longer do. The code now works for all pathing operators, and only has a bug/incompletion wrt a rather rare edge case, so I'm not going to spend any more time on this right now. That time _has_ to be spent on more important things. I've filed this issue, with all the information needed to address it at some later point in time, and if someone else absolutely unequivocally needs this before then, and they want to write a PR: great. Or if they want to pay me to make doing this worth _not_ working on the Primer on Bezier curves, also great.

**Comment by thednp** [comment]
Good luck with that man.

Cheers!

**Comment by thednp** [comment]
Good news: I came back to your code for a spin, it runs perfect on 1k+ SVGs, I hope you don't mind adding your name to the credits list :)

**Comment by thednp** [comment]
> 
> 
> by all means

DONE. Cheers :)

**Comment by Pomax** [comment]
fixed as of 1.2.1
